services/gateway/apis/http-version3/swagger/definitions/blockchainApps.json

```
@@ -66,8 +66,7 @@
 	"blockchainAppsStatsEnvelope": {
 		"type": "object",
 		"required": [
-			"data",
-			"meta"
+			"data"
```

`data` and `meta` must always be present in the response.
```suggestion
			"data",
			"meta"
```